easybuild/easyconfigs/e/ELPA/ELPA-2013.11-ictce-5.3.0.eb

```diff
@@ -23,7 +23,8 @@ sources = ['%(name)s_%(version)s.006_20140312.tar.gz']
 
 patches = ['ELPA_fix-tests.patch']
 
-configopts = '--with-generic-simple --disable-shared FCFLAGS="-I$EBROOTIMKL/mkl/include/intel64/lp64 $FCFLAGS" LIBS="$LIBSCALAPACK"'
+configopts = '--with-generic-simple --disable-shared FCFLAGS="-I$EBROOTIMKL/mkl/include/intel64/lp64 $FCFLAGS" '
+configopts += 'LIBS="$LIBSCALAPACK"'
```

Comment:
maybe include both the `FCFLAGS="..."` and `LIBS="..."` in the same line, and keep the configure flags on the other line?